packages/dso-toolkit/src/components/table/table.mixins.scss

```diff
@@ -2,6 +2,22 @@
 
 @use "../table";
 
+@mixin outside-lines-sides() {
+  thead {
+    border-left: 1px solid table.$th-border-color;
```

Comment:
logical properties.

Geldt voor alle `border`-declaraties in dit document (nog 5 hieronder)